adapters/repos/db/shard_shutdown.go

```diff
@@ -97,6 +103,8 @@ func (s *Shard) Shutdown(ctx context.Context) (err error) {
 			}
 		}
 	} else {
+		err = s.queue.Flush()
+		ec.AddWrap(errors.Wrap(err, "flush vector index queue commitlog"))
```

Comment:
can we here wrap the error without this additional wrap? like so:
```
ec.AddWrap(err, "flush vector index queue commitlog")
```

Reply:
oh good point